src/modules/85-cv/components/ExecutionVerification/components/LogAnalysisContainer/components/ClusterTypeFiltersForLogs.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Checkbox, Container, Layout } from '@harness/uicore'
+import { useStrings } from 'framework/strings'
+import type { ClusterTypeFiltersForLogsProps } from './ClusterTypeFiltersForLogs.types'
+import { getClusterTypes } from '../LogAnalysis.utils'
+import css from './ClusterTypeFiltersForLogs.module.scss'
+
+const ClusterTypeFiltersForLogs: React.FC<ClusterTypeFiltersForLogsProps> = ({
+  totalClustersCount,
+  clusterTypeFilters,
+  onFilterChange
+}) => {
+  const { getString } = useStrings()
+
+  const checkboxItems = getClusterTypes(getString)
+
+  return (
+    <Container className={css.main}>
+      <Layout.Horizontal className={css.filterContainer}>
+        <span data-testid="LogAnalysis_totalClusters">
+          {getString('pipeline.verification.logs.totalClusters')} : {totalClustersCount ?? getString('noData')}
+        </span>
+        <Layout.Horizontal>
+          {checkboxItems.map(item => (
+            <Checkbox
+              key={item.label}
+              label={item.label}
+              value={item.value as string}
+              data-testid={item.label}
+              defaultChecked={clusterTypeFilters?.includes(item.value as any)}
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232318)
```suggestion
              defaultChecked={clusterTypeFilters?.includes(item.value as unknown)}
```